### DISCUSSION TITLE
"static_blocks pass should remove this" what does this mean?

### DISCUSSION THREAD

**Discussion by hahazexia** [swc-project/swc#8958] [Q&A] [answered]
<img width="878" alt="image" src="https://github.com/swc-project/swc/assets/20611129/ada65f0e-6854-41f1-a00b-a6b7f7689bce">

i use nextjs@14.3.0-canary.61 and i give it a try for swc, this is my next.config.js config:

{
  experimental: {
     forceSwcTransforms: true,
  }
}

npm run dev is fine, but when i run build, it failed, here is the message:

<img width="642" alt="image" src="https://github.com/swc-project/swc/assets/20611129/670ac570-1ce0-4fce-9909-c57197f61e11">

and this is corp.ts file:

<img width="857" alt="image" src="https://github.com/swc-project/swc/assets/20611129/149b8b01-02f4-4c96-b424-d66ca0cb0c4a">

**Comment by kdy1** [comment, marked as answer]
It means a bug of SWC. Can you file an issue with a reproduction?


For description, `internal error: entered unreachable code` is the key of it. `internal error` is something not expected, and if it's exposed to the end user, it's always a bug

**Reply by hahazexia** [reply]
Sure, once I have some time, I will immediately provide an example to reproduce the issue.
